compiler/noirc_evaluator/src/ssa.rs

```diff
@@ -85,46 +92,46 @@ pub(crate) fn optimize_into_acir(
 
     let mut ssa = SsaBuilder::new(
         program,
-        options.enable_ssa_logging,
+        options.ssa_logging.clone(),
         options.force_brillig_output,
         options.print_codegen_timings,
         &options.emit_ssa,
     )?
-    .run_pass(Ssa::defunctionalize, "After Defunctionalization:")
-    .run_pass(Ssa::remove_paired_rc, "After Removing Paired rc_inc & rc_decs:")
-    .run_pass(Ssa::separate_runtime, "After Runtime Separation:")
-    .run_pass(Ssa::resolve_is_unconstrained, "After Resolving IsUnconstrained:")
-    .run_pass(|ssa| ssa.inline_functions(options.inliner_aggressiveness), "After Inlining (1st):")
+    .run_pass(Ssa::defunctionalize, "Defunctionalization")
```

Comment:
Instead of removing the "After" prefix and the colon, why not do what the option description says and check if the label _contains_ the pass name rather than _equals_ to it?

Reply:
Oh I see you kept them further down, nice 👍